rust/chains/hyperlane-ethereum/src/rpc_clients/trait_builder.rs

```diff
@@ -24,7 +25,10 @@ use hyperlane_core::{
 };
 
 use crate::signer::Signers;
-use crate::{ConnectionConf, EthereumFallbackProvider, RetryingProvider, RpcConnectionConf};
+use crate::{
+    ConnectionConf, EthereumFallbackProvider, RetryingProvider, RpcConnectionConf,
+    TransactionOverrides,
+};
```

Comment:
Something else to be aware of is that the PendingTransaction we get back and wait for here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/chains/hyperlane-ethereum/src/tx.rs#L56 isn't totally compatible with the gas escalator. If the initial tx is replaced by the gas escalator, in `track_pending_tx` we'll still be looking for the first transaction. I guess this isn't so bad because we'd time out on the initial tx, and our op_submitter logic would eventually realize that the message had been processed, but still things get a bit funky

Reply:
not really something we can fix, the gas escalator unit test also has this issue